invoices/invoiceregistry.go

```diff
@@ -789,13 +796,13 @@ func (i *InvoiceRegistry) NotifyExitHopHtlc(rHash lntypes.Hash,
 
 			i.notifyHodlSubscribers(
 				*NewSettleResolution(
-					invoice.Terms.PaymentPreimage, key, acceptHeight,
+					invoice.Terms.PaymentPreimage, key, acceptHeight, result,
```

Comment:
Check line lengths in this file

Reply:
They look fine to me? Anything sticking out?

Reply:
Maybe your tab size is 4? It should be 8

Reply:
Should it? Contribution guidelines say 4 [here]( Editors should be set to treat a tab as 4 spaces.)?

Reply:
I am sure we had a discussion about that some time ago and that it was def 8